Updated test drive page VsCode instructions (#10859)
flutter/website: src/_includes/docs/install/test-drive/vscode.md

### Create a sample Flutter app {:#create-app-vs-code}

> 1. To use the DevTools from VS Code, install the [Flutter extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter).
>    This will also automatically install the [Dart extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.dart-code).With this extensions, you will be able to easily debug you application.

Reviewer: ```suggestion
1. To use Flutter DevTools from VS Code, install the
    [Flutter extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter).
   This also automatically installs the
   [Dart extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.dart-code).
   With these extensions, you can debug you application.
```
